xeeg_hed_score/dataset_description.json

```diff
@@ -0,0 +1,24 @@
+{
+    "Name": "HED schema library for SCORE annotations example",
+    "BIDSVersion": "1.7.0",
+    "HEDVersion": "8.0.0",
```

Comment:
Needs to have the score version added per pull request in bids specification https://github.com/bids-standard/bids-specification/pull/1106

Reply:
Done